drv/gimlet-seq-server/build.rs

```diff
@@ -12,9 +14,148 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let out = &PathBuf::from(env::var_os("OUT_DIR").unwrap());
     fs::write(out.join("fpga.bin.rle"), compressed)?;
+
+    let disposition = build_i2c::Disposition::Devices;
+
+    if let Err(e) = build_i2c::codegen(disposition) {
+        println!("code generation failed: {}", e);
+        std::process::exit(1);
+    }
+
+    fs::write(out.join("gimlet_regs.rs"), regs()?)?;
+    println!("cargo:rerun-if-changed=fpga.bin");
```

Comment:
Can you move this up to where the fpga.bin is actually being manipulated, to make it easier to find / more obvious?

Reply:
Done